src/components/Header/DonateButton.tsx

```diff
@@ -20,7 +21,7 @@ const DonateButton = ({ header = false, style = {} }) => {
                 ...style,
             }}
         >
-            {header ? t("header:donateButton") : t("home:donateButton")}
+            {t("home:donateButton")}
```

Comment:
question: any reason for removing this logic?

Reply:
We were configuring the text of this button in the local config. After switching to configuring by overwriting the JSONs, I believe this logic is no longer necessary.